src/plugins/condition/components/condition.scss

```diff
@@ -1,8 +1,6 @@
-
-
 .widget-condition {
     background-color: #eee;
-    margin: 0 0 0.6em;
+    margin: 0 0 5px;
```

Comment:
Prefer the use of `em` instead of `px`. Was this change necessary?

Reply:
done